libpod/common_test.go

```diff
@@ -75,7 +75,10 @@ func getTestContainer(id, name, locksDir string) (*Container, error) {
 		valid: true,
 	}
 
-	g := generate.New()
+	g, err := generate.New("linux")
+	if err != nil {
+		return nil, err
+	}
 	ctr.config.Spec = g.Spec()
```

Comment:
This should have `g.Spec()` -> `g.Config` (and similarly in `CreateConfigToOCISpec`).

Reply:
> This should have `g.Spec()` -> `g.Config` (and similarly in `CreateConfigToOCISpec`).

I've filed #1008 for this.